content/getting_started/_index.md

```diff
@@ -9,4 +9,6 @@ Please see the [Using Enzyme](/getting_started/UsingEnzyme) guide for informatio
 
 Please see the [Calling Convention](/getting_started/CallingConvention) guide for detailed information on Enzyme's calling convention and ABI, as well as how to specify how functions should be differentiated.
 
-Please see the [CUDA Guide](/getting_started/CUDAGuide) guide for information on working with CUDA code.
\ No newline at end of file
+Please see the [CUDA Guide](/getting_started/CUDAGuide) guide for information on working with CUDA code.
+
```

Comment:
I think you need to add a Examples tab on the higher level file, otherwise it doesn't show up.

Also the latex isn't getting rendered right now, so we probably need to include a plugin.